lib/Runtime/Language/ReadOnlyDynamicProfileInfo.h

```diff
@@ -50,6 +50,7 @@ namespace Js
             isObjTypeSpecDisabled_jitLoopBody(false),
             isPowIntIntTypeSpecDisabled(false),
             isLoopImplicitCallInfoDisabled(false),
+			isStackArgOptDisabled(false),
```

Comment:
Done.

---
In reply to: [67796330](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67796330) [](ancestors = 67796330)